spec/helpers/perf/drivers/local.lua

```diff
@@ -230,7 +230,7 @@ function _M:generate_flamegraph(title)
 
   local cmds = {
     "/tmp/perf-ost/fix-lua-bt " .. path .. ".bt > " .. path .. ".fbt",
-    "/tmp/perf-fg/stackcollapse.pl " .. path .. ".fbt > " .. path .. ".cbt",
+    "/tmp/perf-fg/stackcollapse-stap.pl " .. path .. ".fbt > " .. path .. ".cbt",
```

Comment:
nitpick: I see some of those commands are repeated in the `terraform` module; would it make sense to factor them into a common module - say, `drivers/constants.lua`?

Reply:
Yeah that's good point, terraform has lots of similarity with local driver. I think we can make them share some operations in the future.